### PR TITLE
K8s worker

### PR DESCRIPTION
**Pull request by Neeraj-Nec**
Provided support of Kubernates for executing FogFlow task

### CODE REVIEW

File: k8sWorker/pod.go [hunk 1]
```diff
@@ -0,0 +1,195 @@
+package main
+
+import (
+        "context"
+        "fmt"
+	"strconv"
+//	"encoding/json"
```
Comment (naveennec) [inline k8sWorker/pod.go:7]: remove the commented code.

File: k8sWorker/pod.go [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+        metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (naveennec) [inline k8sWorker/pod.go:10]: correct the formatting.

File: k8sWorker/pod.go [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+        // _ "k8s.io/client-go/plugin/pkg/client/auth/openstack"
+)
+
+type pod struct{
+	client        *docker.Client
+}
+
+func (p *pod) pod(dockerImage string, port string, adminCfg []interface{}) (string, error){
+
+	// prepare the configuration for a docker container, host mode for the container network
```
Comment (naveennec) [inline k8sWorker/pod.go:38]: remove the debugging code.

File: k8sWorker/pod.go [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+			}
+			if key == "id" {
+				id = value.(string)
+			}
+			if key == "type"{
+				Etype = value.(string)
+			}
+		}
+	}
+//	fmt.Println("************line #46 b[i] is**********",b)
```
Comment (naveennec) [inline k8sWorker/pod.go:66]: remove the debugging code.

File: k8sWorker/startworkers.sh [hunk 5]
```diff
@@ -0,0 +1,13 @@
+#echo "start worker #1"
+#nohup ./worker -f config/w1.json & 
+
+#echo "start worker #2"
+#nohup ./worker -f config/w2.json & 
+
```
Comment (naveennec) [inline k8sWorker/startworkers.sh:6]: Remove the commented code.

**Review by naveennec — CHANGES_REQUESTED** [review]

**Comment by Neeraj-Nec** [comment]
incorporated commnt in k8sWorker branch

**Pull request closed without merging**